### PR TITLE
fix: splash screen slightly jumps

### LINKED ISSUE #2542: [hold for payment 7 May] Loading screen logo slightly shifts up before fading away.
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Expected Result:
Loading screen logo is stable. 

## Actual Result:
Loading screen logo slightly shifts up before fading away.

## Action Performed:
1. Kill the chat app if open 
2. Relaunch the app
## Workaround:
No need for workaround. Visual issue. 

## Platform:
Where is this issue occurring?

 Web
 iOS
 Android ✔️ 
 Desktop App
 Mobile Web

**Version Number:** 1.0.29-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/44479856/115789564-55352b80-a393-11eb-92f7-1f7e60325dbe.mp4

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

___ 
From @quinthar https://expensify.slack.com/archives/C01GTK53T8Q/p1619062191006900

>ISSUE: the new loading screen starts with the icon in the middle, but then the shifts ever so slightly up right before disappearing. It reliably does this every time I "cold open" the app on Android

### PR DESCRIPTION
**Pull request by parasharrajat**
Please review @Beamanator 

### Details
A detailed proposal can be found here https://github.com/Expensify/Expensify.cash/issues/2542#issuecomment-826450366.

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes #2542 

### Tests / QA Steps
1. Kill the chat app if open
2. Relaunch the app
3. Notice that the Splash screen should fade away without jumping up or down.

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [x] Android (affected platform)

#### Android

 

<details open>
<summary><h3>Issue</h3></summary>

https://user-images.githubusercontent.com/24370807/116077997-bae72780-a6b3-11eb-87d0-2f40c41d3e5b.mp4

</details>
<details open>
<summary><h3> After Fix </h3></summary>

https://user-images.githubusercontent.com/24370807/116078209-f2ee6a80-a6b3-11eb-9e15-09687bb76780.mp4

</details>

### CODE REVIEW

**Review by Beamanator — APPROVED** [review]
Tested and looks great 👍 Thanks @parasharrajat - will wait on one more reviewer before merge

**Review by Dal-Papa — APPROVED** [review]

**Comment by roryabraham** [comment]
@parasharrajat, this PR caused [a regression](https://github.com/Expensify/Expensify.cash/issues/2577) on Android. Reverting this PR [here](https://github.com/Expensify/Expensify.cash/pull/2581), you'll need to resubmit a new solution to the [fixed issue](https://github.com/Expensify/Expensify.cash/issues/2542).

**Pull request merged**
